src/main/java/cat/udl/eps/softarch/domain/User.java

```diff
@@ -29,11 +30,23 @@
     private String email;
     private Date birthday;
     private String country;
+    @NotEmpty(message = "Is necessary a password")
```

Comment:
Password is not mandatory for OAuth logins afaik. If you force them to have a password, they might not be able to log in.